README.md

```diff
@@ -79,6 +79,24 @@ import XMTP
 let client = try await Client.create(account: account)
 ```
 
+### Creating a client from saved keys
+
+You can save your keys from the client via the `v1keys` property:
+
+```swift
+// Create the client with a `SigningKey` from your app
+let client = try await Client.create(account: account)
```

Comment:
Suggestion: It might be worth adding the environment to the example because we've run into many issues in the past where people forget dev keys != prod keys.